R/r.R

```diff
@@ -15,38 +18,42 @@ use_r <- function(name = NULL) {
 }
 
 check_file_name <- function(name) {
-  if (!valid_file_name(name)) {
+  if (!valid_file_name(path_ext_remove(name))) {
     stop(
       value(name), " is not a valid file name. It should:\n",
       "* Contain only ASCII letters, numbers, '-', and '_'\n",
       call. = FALSE
     )
   }
+  name
 }
 
 valid_file_name <- function(x) {
   grepl("^[[:alnum:]_-]+$", x)
 }
 
-find_r_name <- function(name = NULL) {
-  if (!is.null(name)) {
-    check_file_name(name)
-    return(slug(name, ".R"))
-  }
-
+get_active_r_file <- function(path = "R") {
   if (!rstudioapi::isAvailable()) {
-    stop("Argument ", code("name"), " is missing, with no default", call. = FALSE)
+    stop(glue("Argument {code('name')} must be specified."), call. = FALSE)
```

Comment:
It feels like we are doing a lot of `stop(glue(` etc, maybe worth creating a helper function for it, e.g. something like https://github.com/r-lib/fs/blob/6ee3322076bd2cb6b9cacd0b52ac1717b11996e4/R/utils.R#L113-L123.

Or we can wait on this until things are more sure in rlang.

Reply:
Yeah, I will use glue more, but in another PR.